parm/config/sfs/config.resources

```diff
@@ -0,0 +1,371 @@
+#! /usr/bin/env bash
+
+########## config.resources ##########
+# Set resource information for job tasks
+# e.g. walltime, node, cores per node, memory etc.
+
+if (( $# != 1 )); then
+
+    echo "Must specify an input task argument to set resource variables!"
+    exit 1
+
+fi
+
+step=$1
+
+echo "BEGIN: config.resources"
+
+case ${machine} in
+  "WCOSS2")   max_tasks_per_node=128;;
+  "HERA")     max_tasks_per_node=40;;
+  "ORION")    max_tasks_per_node=40;;
+  "HERCULES") max_tasks_per_node=80;;
+  "JET")
+    case ${PARTITION_BATCH} in
+      "xjet")          max_tasks_per_node=24;;
+      "vjet" | "sjet") max_tasks_per_node=16;;
+      "kjet")          max_tasks_per_node=40;;
+      *)
+        echo "FATAL ERROR: Unknown partition ${PARTITION_BATCH} specified for ${machine}"
+        exit 3
+    esac
+    ;;
+  "S4")
+    case ${PARTITION_BATCH} in
+      "s4")  max_tasks_per_node=32;;
+      "ivy") max_tasks_per_node=20;;
+      *)
+        echo "FATAL ERROR: Unknown partition ${PARTITION_BATCH} specified for ${machine}"
+        exit 3
+    esac
+    ;;
+  "AWSPW")
+    export PARTITION_BATCH="compute"
+    max_tasks_per_node=48
+    ;;
+  "AZUREPW")
+    export PARTITION_BATCH="compute"
+    max_tasks_per_node=36
```

Comment:
A thought for another time: it might be nice to have a common resources file for machine parameters and tasks that run for all `NET`s.

Reply:
Opened issue https://github.com/NOAA-EMC/global-workflow/issues/3336 to look into this.  Resolving this conversation.